source/ranges.tex

```diff
@@ -1584,7 +1584,7 @@
 
 \begin{itemdecl}
 template<class R>
-  inline constexpr bool @\exposidnc{is-initializer-list}@ = @\seebelow@;        // \expos
+  constexpr bool @\exposidnc{is-initializer-list}@ = @\seebelow@;        // \expos
```

Comment:
```suggestion
  constexpr bool @\exposidnc{is-initializer-list}@ = @\seebelow@;                // \expos
```
Just in case it already was aligned. The check script skips these.

Reply:
I hope I fixed it correctly.

Reply:
Seems like you added one too many.